filebeat/input/journald/pkg/journalctl/testdata/corner-cases.json

```diff
@@ -0,0 +1,96 @@
+  "COREDUMP_COMM": "kdeconnectd",
+  "COREDUMP_ROOT": "/",
+  "__CURSOR": "s=e82795fad4ce42b79fb3da0866d91f7e;i=4b34d3;b=62c460d8931346bf85d24dfe22d2d2fb;m=41ffc01;t=61c2e1d5b065f;x=dff9cffe8a25c611",
+  "_SYSTEMD_CGROUP": "/system.slice/system-systemd\\x2dcoredump.slice/systemd-coredump@0-3732-0.service",
+  "__SEQNUM": "4928723",
+  "COREDUMP_CGROUP": "/user.slice/user-1000.slice/user@1000.service/app.slice/app-dbus\\x2d:1.9\\x2dorg.kde.kdeconnect.slice/dbus-:1.9-org.kde.kdeconnect@0.service",
+  "COREDUMP_PROC_CGROUP": "0::/user.slice/user-1000.slice/user@1000.service/app.slice/app-dbus\\x2d:1.9\\x2dorg.kde.kdeconnect.slice/dbus-:1.9-org.kde.kdeconnect@0.service\n",
+  "COREDUMP_PACKAGE_JSON": "{\"elfType\":\"coredump\",\"elfArchitecture\":\"AMD x86-64\"}",
+  "_TRANSPORT": "journal",
+  "COREDUMP_UID": "1000",
+  "_CAP_EFFECTIVE": "0",
+  "__MONOTONIC_TIMESTAMP": "69204993",
+  "COREDUMP_SIGNAL_NAME": "SIGABRT",
+  "COREDUMP_PROC_STATUS": "Name:\tkdeconnectd\nUmask:\t0022\nState:\tS (sleeping)\nTgid:\t3717\nNgid:\t0\nPid:\t3717\nPPid:\t1998\nTracerPid:\t0\nUid:\t1000\t1000\t1000\t1000\nGid:\t1000\t1000\t1000\t1000\nFDSize:\t64\nGroups:\t108 972 986 998 1000 1001 1002 \nNStgid:\t3717\nNSpid:\t3717\nNSpgid:\t3717\nNSsid:\t3717\nKthread:\t0\nVmPeak:\t  243788 kB\nVmSize:\t  178252 kB\nVmLck:\t       0 kB\nVmPin:\t       0 kB\nVmHWM:\t   27476 kB\nVmRSS:\t   27476 kB\nRssAnon:\t    3328 kB\nRssFile:\t   24148 kB\nRssShmem:\t       0 kB\nVmData:\t    9844 kB\nVmStk:\t     132 kB\nVmExe:\t       8 kB\nVmLib:\t   43268 kB\nVmPTE:\t     192 kB\nVmSwap:\t       0 kB\nHugetlbPages:\t       0 kB\nCoreDumping:\t1\nTHP_enabled:\t1\nuntag_mask:\t0xffffffffffffffff\nThreads:\t2\nSigQ:\t1/255932\nSigPnd:\t0000000000000000\nShdPnd:\t0000000000000000\nSigBlk:\t0000000000000000\nSigIgn:\t0000000000001000\nSigCgt:\t0000000100000000\nCapInh:\t0000000000000000\nCapPrm:\t0000000000000000\nCapEff:\t0000000000000000\nCapBnd:\t000001ffffffffff\nCapAmb:\t0000000000000000\nNoNewPrivs:\t0\nSeccomp:\t0\nSeccomp_filters:\t0\nSpeculation_Store_Bypass:\tthread vulnerable\nSpeculationIndirectBranch:\tconditional enabled\nCpus_allowed:\tffff\nCpus_allowed_list:\t0-15\nMems_allowed:\t00000001\nMems_allowed_list:\t0\nvoluntary_ctxt_switches:\t9\nnonvoluntary_ctxt_switches:\t3\nx86_Thread_features:\t\nx86_Thread_features_locked:\t\n",
+  "_PID": "3740",
+  "COREDUMP_HOSTNAME": "millennium-falcon",
+  "_BOOT_ID": "62c460d8931346bf85d24dfe22d2d2fb",
+  "_HOSTNAME": "millennium-falcon",
+  "COREDUMP_OWNER_UID": "1000"
+},
+{
+  "CODE_FILE": "../pulseaudio/src/modules/bluetooth/backend-native.c",
+  "CODE_FUNC": "hfp_rfcomm_handle",
+  "CODE_LINE": "658",
+  "MESSAGE": [85,110,97,98,108,101,32,116,111,32,112,97,114,115,101,32,105,110,100,105,99,97,116,111,114,32,111,102,32,65,84,43,66,73,65,32,99,111,109,109,97,110,100,58,32,65,84,43,66,73,65,61,49,44,49,44,48,44,48,44,48,13,65,84,43,66,67,83,61,50],
```

Comment:
That's an oddly interesting case, [Journald documentation](https://www.man7.org/linux/man-pages/man7/systemd.journal-fields.7.html) states:
> Entries in the journal (as written by
       [systemd-journald.service(8)](https://www.man7.org/linux/man-pages/man8/systemd-journald.service.8.html)) resemble a UNIX process environment
       block in syntax but with field values that may include binary
       data, and with non-unique field names permitted. Primarily, field
       values are formatted UTF-8 text strings — binary encoding is used
       only where formatting as UTF-8 text strings makes little sense.
       New fields may freely be defined by applications, but a few
       fields have special meanings, which are listed below. Typically,
       fields may only appear once per log entry, however there are
       special exceptions: some fields may appear more than once per
       entry, in which case this is explicitly mentioned below. Even
       though the logging subsystem makes no restrictions on which
       fields to accept non-unique values for, it is strongly
       recommended to avoid relying on this for the fields listed below
       (except where listed otherwise, as mentioned) in order to avoid
       unnecessary incompatibilities with other applications.

However [their protocol](https://systemd.io/JOURNAL_NATIVE_PROTOCOL/) (see 'Data Format') allows to encode any field as binary data.

On this specific case, if I just try to convert those number to rune, then string the conversion works, but there seems to be some control characters there that get interpreted by the terminal and the output is not a clean string. Interestingly enough the [Go Playground](https://go.dev/play/p/jqHr8Bdv1OZ) manages to print the string, it's not a terminal, so it's likely handing control characters differently.

Go Playground output:
```
Unable to parse indicator of AT+BIA command: AT+BIA=1,1,0,0,0
AT+BCS=2
```

My terminal output:
```                   
AT+BCS=2
```

Reading it with the default output of `journalctl` just prints that the message is a blob of data:
```
 journalctl --cursor="s=e82795fad4ce42b79fb3da0866d91f7e;i=4b454d;b=62c460d8931346bf85d24dfe22d2d2fb;m=39c3b04e4;t=61c33f69f4665;x=8dd4baba5585d42b" -n 1 _EXE=/usr/bin/pulseaudio                                          
Jul 01 14:42:46 millennium-falcon pulseaudio[3728]: [70B blob data]
```

If I use the JSON output, the result is this JSON where I'm commenting.